switchStatement.js

```diff
@@ -39,3 +82,25 @@ Example:
 *Note: Check the file called "Clothing_Recommendations.md" to find the list of weather forecasts and their corresponding clothing recommendations.*  
 ************************************************************************************************/
 //  TODO: ADD YOUR CODE BELOW
+let weather=`sunny`
+switch (weather) {
+  case `sunny`:console.log(`What's the weather forecast for today? Sunny 
+  It is recommended to wear light, colorful clothing to avoid the sun's rays`);
+    
+    break;
+
+    case `coludy`:console.log(`What's the weather forecast for today?cooludy
+    It is recommended to wear thick clothes `);
+
+    break
+    case `rainy`:console.log(`What's the weather forecast for today?rainy
+    It is recommended to take an umbrella when going out `);
+    break
+    case `snowy`:console.log(`What's the weather forecast for today?snowey
+    It is recommended to wear snowresistant clothing`);
+    break
+  default:console.log("Sorry, we do not have recommendations for that weather condition.");
+    ;
+}
```

Comment:
Please organize your code better next time:
- remove the extra spaces between line 89 and 90, and 93 and 95